CHANGES.rst

```diff
@@ -3,9 +3,108 @@
 0.14 (unreleased)
 -----------------
 
-- Planned for September 15, 2019
+Summary
++++++++
+
+- Released on September 30th, 2019
+- 8 contributors
+- 92 pull requests (not all listed below)
+
+**What's new**
+
+Gammapy v0.14 features a new high level analysis interface. Starting from
+a YAML configuration file, it supports the standard use-cases of joined
+or stacked 3D as well as 1D reflected region analyses. It also supports
+computation of flux points for all cases. The usage of this new ``Analysis``
+class is demonstrated in the ``hess.ipynb`` tutorial.
+
+To bundle all model classes and the functionality from ``gammapy.utils.fitting``
+into a single namespace, a new ``gammapy.modeling`` subpackage was introduced.
+All spectral, spatial, temporal and combined models were moved to the
+``gammapy.modeling`` subpackage and renamed following a consistent naming
+scheme. A new ``ConstantTemporalModel`` was introduced and the ``GaussianSpatialModel``
+now supports elongation parameters. A bug that lead to an incorrect flux
+normalisation of the ``PointSpatialModel`` at high latitudes was fixed.
+
+A new ``MapDataset.to_spectrum_dataset()`` method allows to reduce a map
+dataset to a spectrum dataset in a specified analysis region.
+
```

Comment:
You could add here the removal of `SpectrumDatasetOnOffStacker` and the replacement with `.stack()` methods on each dataset type as well as the `Datasets.stack_reduce()` method to stack a list of datasets sharing the same type and geometry.

Reply:
Done